cypress/e2e/tests/migration/applicationinventory/assessment/miscellaneous.test.ts

```diff
@@ -231,6 +231,30 @@ describe(["@tier3"], "Tests related to application assessment and review", () =>
         archetypes[0].delete();
         deleteByList(tags);
     });
+    it("Deletes assessments from archived questionnaire associated with an archetype and an application", function () {
```

Comment:
Could you add a line to separate the two tests ?

Reply:
done